<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QM_Pot_GUI</class>
 <widget class="QMainWindow" name="QM_Pot_GUI">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QM_Pot_GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="infinite">
       <attribute name="title">
        <string>Infinite</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QWidget" name="infinite_widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>270</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>270</width>
            <height>16777215</height>
           </size>
          </property>
          <widget class="QLabel" name="infinite_width_label">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>54</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Width</string>
           </property>
          </widget>
          <widget class="QSlider" name="infinite_width_slide">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>160</width>
             <height>18</height>
            </rect>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>40</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="pageStep">
            <number>4</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
          </widget>
          <widget class="QSpinBox" name="infinite_width_spin">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>35</y>
             <width>62</width>
             <height>27</height>
            </rect>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>40</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
          </widget>
          <widget class="QWidget" name="infinite_mode" native="true">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>80</y>
             <width>250</width>
             <height>61</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <widget class="QLabel" name="infinite_mode_label">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>54</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Mode</string>
            </property>
           </widget>
           <widget class="QSlider" name="infinite_mode_slide">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>160</width>
              <height>18</height>
             </rect>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
           <widget class="QSpinBox" name="infinite_mode_spin">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>25</y>
              <width>62</width>
              <height>27</height>
             </rect>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
           </widget>
          </widget>
          <widget class="QListWidget" name="infinite_mode_list">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>150</y>
             <width>231</width>
             <height>192</height>
            </rect>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::SelectedClicked</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
           <item>
            <property name="text">
             <string>1. Mode</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2. Mode</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3. Mode</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4. Mode</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5. Mode</string>
            </property>
           </item>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QCustomPlot" name="infinite_graph" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="finite">
       <attribute name="title">
        <string>Finite</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QWidget" name="finite_widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>270</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>270</width>
            <height>16777215</height>
           </size>
          </property>
          <widget class="QWidget" name="finite_mass" native="true">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>150</y>
             <width>250</width>
             <height>61</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <widget class="QLabel" name="finite_mass_label">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>54</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Mass</string>
            </property>
           </widget>
           <widget class="QSlider" name="finite_mass_slide">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>160</width>
              <height>18</height>
             </rect>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>250</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
           <widget class="QSpinBox" name="finite_mass_spin">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>25</y>
              <width>62</width>
              <height>27</height>
             </rect>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>250</number>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="finite_depth" native="true">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>80</y>
             <width>250</width>
             <height>61</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <widget class="QLabel" name="finite_depth_label">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>54</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Depth</string>
            </property>
           </widget>
           <widget class="QSlider" name="finite_depth_slide">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>160</width>
              <height>18</height>
             </rect>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
           <widget class="QSpinBox" name="finite_depth_spin">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>25</y>
              <width>62</width>
              <height>27</height>
             </rect>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="finite_width" native="true">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>250</width>
             <height>61</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <widget class="QLabel" name="finite_width_label">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>54</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Width</string>
            </property>
           </widget>
           <widget class="QSlider" name="finite_width_slide">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>160</width>
              <height>18</height>
             </rect>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>75</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
           <widget class="QSpinBox" name="finite_width_spin">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>25</y>
              <width>62</width>
              <height>27</height>
             </rect>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>75</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QCustomPlot" name="finite_graph" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="harmonic">
       <attribute name="title">
        <string>Harmonic</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QWidget" name="harmonic_widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>270</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>270</width>
            <height>16777215</height>
           </size>
          </property>
          <widget class="QWidget" name="harmonic_mass" native="true">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>80</y>
             <width>250</width>
             <height>61</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <widget class="QLabel" name="harmonic_mass_label">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>54</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Mass</string>
            </property>
           </widget>
           <widget class="QSlider" name="harmonic_mass_slide">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>160</width>
              <height>18</height>
             </rect>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>250</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
           <widget class="QSpinBox" name="harmonic_mass_spin">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>25</y>
              <width>62</width>
              <height>27</height>
             </rect>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>250</number>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="harmonic_frequ" native="true">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>250</width>
             <height>61</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <widget class="QLabel" name="harmonic_frequ_label">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>54</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Width</string>
            </property>
           </widget>
           <widget class="QSlider" name="harmonic_frequ_slide">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>160</width>
              <height>18</height>
             </rect>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>75</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
           <widget class="QSpinBox" name="harmonic_frequ_spin">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>25</y>
              <width>62</width>
              <height>27</height>
             </rect>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>75</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QCustomPlot" name="harmonic_graph" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="coulomb">
       <attribute name="title">
        <string>Coulomb</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuFigure">
    <property name="title">
     <string>Figure</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionPDF"/>
     <addaction name="actionPNG"/>
    </widget>
    <addaction name="menuExport"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <addaction name="actionProbability"/>
    <addaction name="actionMultiple_Modes"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuFigure"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPDF">
   <property name="text">
    <string>PDF</string>
   </property>
  </action>
  <action name="actionPNG">
   <property name="text">
    <string>PNG</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionProbability">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Probability</string>
   </property>
  </action>
  <action name="actionMultiple_Modes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Multiple Modes</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
